apps/design/frontend/src/features_context.tsx

```diff
@@ -69,6 +69,11 @@ enum UserFeature {
    * Allow the user to select ballot languages.
    */
   BALLOT_LANGUAGE_CONFIG = 'BALLOT_LANGUAGE_CONFIG',
+  /**
+   * Show the user a toggle that allows them to include audio files in
+   * ballot and election package export.
+   */
+  EXPORT_AUDIO_TOGGLE = 'EXPORT_AUDIO_TOGGLE',
```

Comment:
Do we need a feature flag for this? Seems like it's part of the feature of accessing the export screen

Reply:
I thought it might be useful if we eventually do a general release of the export page, but we can simplify now and cross that bridge when we get there.